### DISCUSSION TITLE
Text in polygon

### DISCUSSION THREAD

**Discussion by Innovdata-damien** [maplibre/maplibre-gl-js#1728] [Q&A]
Hello ! 😄 i wan't to know if it possible to add a text, alway visible, who fit the size of the parent  polygon

**Comment by teifler** [comment]
Did you found a way to add text? Size wouldn't matter for me

**Comment by AnsariAffan007** [comment]
Hi @Innovdata-damien , 
This is quite late, but did you find any solution to this problem ?
